test/system/irohad_test.cpp

```diff
@@ -67,20 +67,18 @@ class IrohadTest : public AcceptanceFixture {
   }
 
   void launchIroha(const std::string &parameters) {
-    output_ = boost::process::ipstream();
-    reader_ = std::thread([this] {
-        std::string line;
-        while (std::getline(this->output_, line)) {
-          if (line.find("iroha initialized") != std::string::npos) {
-            return;
-          }
-        }
-
-    });
-    iroha_process_.emplace(irohad_executable.string() + parameters, boost::process::std_out > output_);
-    reader_.join();
+    iroha_process_.emplace(irohad_executable.string() + parameters);
+    auto channel = grpc::CreateChannel(kAddress + ":" + std::to_string(kPort),
```

Comment:
Should be internal port, not torii. Otherwise the first one may have an issue that is not covered

Reply:
Since the transaction are sent in some tests, this way is fine (and probably even better), resolved